apps/builder/src/locales/pt.ts

```diff
@@ -21,4 +25,142 @@ export default defineLocale({
+  'account.preferences.graphNavigation.mouse.label': 'Mouse',
+  'account.preferences.graphNavigation.mouse.description':
+    'Mova arrastando o quadro e amplie/reduza usando a roda de rolagem',
+  'account.preferences.graphNavigation.trackpad.label': 'Trackpad',
+  'account.preferences.graphNavigation.trackpad.description':
+    'Mova o quadro usando 2 dedos e amplie/reduza fazendo pinça',
+  'account.preferences.appearance.heading': 'Aparência',
+  'account.preferences.appearance.systemLabel': 'Sistema',
+  'account.preferences.appearance.lightLabel': 'Claro',
+  'account.preferences.appearance.darkLabel': 'Escuro',
+  'account.myAccount.changePhotoButton.label': 'Alterar foto',
+  'account.myAccount.changePhotoButton.specification':
+    '.jpg ou.png, máximo 1MB',
+  'account.myAccount.emailInput.disabledTooltip':
+    'A atualização do e-mail não está disponível. Entre em contato com o suporte se quiser alterá-lo.',
+  'account.myAccount.emailInput.label': 'Endereço de e-mail:',
+  'account.myAccount.nameInput.label': 'Nome:',
+  'analytics.viewsLabel': 'Visualizações',
+  'analytics.startsLabel': 'Inícios',
+  'analytics.completionRateLabel': 'Taxa de conclusão',
+  'auth.signin.heading': 'Entrar',
+  'auth.signin.noAccountLabel.preLink': 'Não tem uma conta?',
+  'auth.signin.noAccountLabel.link': 'Registe-se gratuitamente',
+  'auth.register.heading': 'Criar uma conta',
+  'auth.register.alreadyHaveAccountLabel.preLink': 'Já tem uma conta?',
+  'auth.register.alreadyHaveAccountLabel.link': 'Entrar',
+  'auth.error.default': 'Tente entrar com uma conta diferente.',
+  'auth.error.email':
+    'E-mail não encontrado. Tente entrar com um fornecedor diferente.',
+  'auth.error.oauthNotLinked':
+    'Para confirmar sua identidade, entre com a mesma conta que você usou originalmente.',
+  'auth.error.unknown': 'Ocorreu um erro. Tente novamente.',
+  'auth.signinErrorToast.title': 'Não autorizado',
+  'auth.signinErrorToast.description': 'As inscrições estão desativadas.',
+  'auth.noProvider.preLink': 'Você precisa',
+  'auth.noProvider.link':
+    'configurar pelo menos um fornecedor de autenticação (E-mail, Google, GitHub, Facebook ou Azure AD).',
+  'auth.orEmailLabel': 'Ou com seu email',
+  'auth.emailSubmitButton.label': 'Enviar',
+  'auth.magicLink.title': 'Um email de link mágico foi enviado. 🪄',
+  'auth.magicLink.description': 'Certifique-se de verificar sua pasta de spam.',
+  'auth.socialLogin.githubButton.label': 'Continuar com GitHub',
+  'auth.socialLogin.googleButton.label': 'Continuar com Google',
+  'auth.socialLogin.facebookButton.label': 'Continuar com Facebook',
+  'auth.socialLogin.azureButton.label': 'Continuar com {azureProviderName}',
+  'auth.socialLogin.gitlabButton.label': 'Continuar com {gitlabProviderName}',
+  'auth.socialLogin.customButton.label': 'Continuar com {customProviderName}',
+  'billing.billingPortalButton.label': 'Portal de faturação',
```

Comment:
In pt-BR, the word `faturação` is not much used. The word `cobrança` is closer to en-US word `billing`.